x-pack/plugins/ml/public/file_datavisualizer/components/file_contents/file_contents.js

```diff
@@ -22,27 +22,36 @@ export function FileContents({ data, format, numberOfLines }) {
   }
 
   const formattedData = limitByNumberOfLines(data, numberOfLines);
+  const fileContentsHeader = i18n.translate('xpack.ml.fileDatavisualizer.fileContents.fileContentsTitle', {
+    defaultMessage: `File contents`
+  });
+  const fileContentsDescription = i18n.translate('xpack.ml.fileDatavisualizer.fileContents.firstLinesDescription', {
+    defaultMessage: `First {numberOfLines, plural, zero {# line} one {# line} other {# lines}}`,
+    values: {
+      numberOfLines
+    }
+  });
+  const ariaLabelI18n = i18n.translate('xpack.ml.fileDatavisualizer.fileContents.ariaLabelBeforeSentUserToCodeEditor', {
+    defaultMessage: `{translatedHeader}. {translatedAmountOfFirstLines} is shown in a code editor below. Hit Tab key to go to code editor.`,
```

Comment:
I wouldn't use labels as parameters here (`translatedHeader`, `translatedAmountOfFirstLines`). Because it's not clear that they are localized labels too, and, what it's more important, the localization for `translatedHeader` and `translatedAmountOfFirstLines` can be different taking into account the context that they have in this message. I suggest specifying the whole message here:
```js
i18n.translate('xpack.ml.fileDatavisualizer.fileContents.beforeSendingUserToCodeEditorAriaLabel', {
  defaultMessage: 'File contents. First {numberOfLines, plural, one {# line is} other {# lines are}} shown in a code editor below. Hit Tab key to go to code editor.',
  values: {
      numberOfLines
     }
```

Otherwise we should add the description with message IDs of `translatedHeader` and `translatedAmountOfFirstLines` to the vendors can see that these parameters are also localizable.